frontend/apps/docs/content/docs/parser/supported-formats/rails.mdx

```diff
@@ -2,4 +2,115 @@
 title: schema.rb (Ruby on Rails)
 ---
 
-TBD
+If you’re using Ruby on Rails with ActiveRecord, in most cases you can automatically generate a useful ER diagram with Liam ERD. This page provides instructions and tips for generating an ER diagram in a Rails project.
+
+## ActiveRecord and db/schema.rb
+
+When a Rails application manages migrations using ActiveRecord, the latest database structure is typically described in `db/schema.rb`.  
+Since `db/schema.rb` is recommended to be kept under version control, most projects will have it in their Git repository.
+
+When using Liam CLI, specify `--format schemarb` and `--input db/schema.rb` as follows:
+
+<Tabs items={["npm", "pnpm", "yarn", "bun"]}>
+  <Tab value="npm" copyable>npx @liam-hq/cli erd build --format schemarb --input db/schema.rb</Tab>
+  <Tab value="pnpm" copyable>pnpm dlx @liam-hq/cli erd build --format schemarb --input db/schema.rb</Tab>
+  <Tab value="yarn" copyable>yarn dlx @liam-hq/cli erd build --format schemarb --input db/schema.rb</Tab>
+  <Tab value="bun" copyable>bun x @liam-hq/cli erd build --format schemarb --input db/schema.rb</Tab>
+</Tabs>
+
+If the above command runs without issue, you should see an ER diagram generated.
+
+## Under the Hood
+
+- Liam CLI does not run an internal Ruby or Rails runtime. In other words, there is no Ruby process running under the hood.
+- Instead, it analyzes the content of `db/schema.rb` using the [`ruby/prisma` parser](https://github.com/ruby/prism).
+
+## If You Don’t Have `db/schema.rb` but Use `db/structure.sql`
+
+If your Rails app doesn’t have `db/schema.rb` and instead uses `db/structure.sql`, here’s what you need to know.
+
+### For Rails 7.0 or Later
+
+Somewhere under `config` (most commonly `config/application.rb`), you may have `config.active_record.schema_format = :sql`.  
+In such cases, running the dump command `rails db:schema:dump` will generate `db/structure.sql`.
+
+If you want to obtain `db/schema.rb`, run the following in an environment with database access:
+
+- Change `config.active_record.schema_format` to `:ruby` and run `rails db:schema:dump`.
+- Alternatively, set the environment variable `SCHEMA_FORMAT=ruby` and run `rails db:schema:dump`.
+  - For example: `SCHEMA_FORMAT=ruby rails db:schema:dump`
```

Comment:
I haven't actually created a sample project to verify it yet, but it should be correct, so I'd like to merge it into the main branch. I'll check the correctness later today.